.buildkite/scripts/pipelines/security_solution_quality_gate/edr_workflows/mki_security_solution_defend_workflows_osquery.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [ -z "$1" ]
+  then
+    echo "No target script from the package.json file, is supplied"
+    exit 1
+fi
+
+
+source .buildkite/scripts/common/util.sh
+.buildkite/scripts/bootstrap.sh
+
+export JOB=kibana-defend-workflows-osquery-serverless-cypress
+
+buildkite-agent meta-data set "${BUILDKITE_JOB_ID}_is_test_execution_step" "true"
+
+source .buildkite/scripts/pipelines/security_solution_quality_gate/prepare_vault_entries.sh
+
+cd x-pack/plugins/osquery
+set +e
```

Comment:
nit: can we move this below vault_get?  we probably want to exit early if vault returns an error.

Reply:
Sure, however, I'm not too well-versed in these configs. Could you help me out by suggesting where it could be placed?